clinica/iotools/converters/nifd_to_bids/nifd_utils.py

```diff
@@ -26,7 +26,7 @@ def find_clinical_data(clinical_data_directory: PathLike) -> Optional[DataFrame]
 
 
 def read_clinical_data(clinical_data_directory: PathLike) -> DataFrame:
-    from pandas import NA, CategoricalDtype
+    import pandas as pd
```

Comment:
Why are these changes in this PR?

Reply:
Now that we are using pandas v1.2, it could also be simplified with `dtype="Int64"` and `dtype="Float64"` (notice the capitalization to distinguish between Pandas and Numpy).